docs/docs/examples/repl.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: docs
+title: REPL
+section: examples
+sort_order: 1001
+---
+
+REPL
+----
+
+This examples shows how to run OPA as an interactive shell or [REPL (read-eval-print loop)](https://en.wikipedia.org/wiki/Read–eval–print_loop).
+
+REPLs are great for learning new languages and running quick experiments. You can take advantage of OPA's REPL to quickly
+run ad-hoc queries or prototype policies.
+
+In this example, we will use the REPL to define rules to identify servers that violate a security policy: servers exposing (unencrypted) HTTP must not be connected to a public network.
+
+Once you finish this example, you will be familiar with:
+
+- Running OPA as an interactive shell/REPL.
+- Writing ad-hoc queries in [Rego](/docs/lang.html).
+
+Prerequisites
+=============
+
+- This example requires that you have the latest version of OPA. You can download the latest version of OPA at [openpolicyagent.org](/index.html)
+
+- This example assumes you have the OPA executable in your $PATH.
+
+Steps
+=====
+
+1. First, make sure you can run the REPL on your machine:
+
+        opa run
+
+    Without any data, you can experiment with simple boolean expressions to get the hang of it:
+
```

Comment:
I feel we could build this up a bit more--break it into smaller pieces.  This REPL isn't like anything anyone has seen before.  The explanation below is good.  But I feel some more intro material would be helpful.